obvs/patchscope.py

```diff
@@ -55,6 +56,7 @@ class SourceContext:
     prompt: str | torch.Tensor
     position: Sequence[int] | None = None
     layer: int = -1
+    head: int | None = None
```

Comment:
Should we accept `[int]` for `head`, similar to `prompt`?

Reply:
Good point, we should definitely accept Sequence[int] for the heads as well -> will change

Reply:
This is already supported, I just changed the type hint